rsocket-core/src/main/java/io/rsocket/client/KeepAliveProvider.java

```diff
@@ -29,20 +30,20 @@
  */
 public final class KeepAliveProvider {
 
-    private volatile boolean ackThresholdBreached;
+    private volatile ConnectionException ackThresholdBreached;
     private volatile long lastKeepAliveMillis;
     private volatile long lastAckMillis;
-    private final Flux<Long> ticks;
+    private final Flux<Object> ticks;
```

Comment:
The numbers are only used in tests, nowhere else.  The wallclock spacing between ticks is the interesting part, not the value emitted each time it ticks.  I'll make this clearer in the documentation.